src/tools/publish_toolstate.py

```diff
@@ -41,7 +41,7 @@
         'ryankurte', 'thejpster', 'therealprof',
     },
     'edition-guide': {'ehuss', 'Centril', 'steveklabnik'},
-    'rustc-guide': {'mark-i-m', 'spastorino', 'amanjeev', 'JohnTitor'},
+    'rustc-dev-guide': {'mark-i-m', 'spastorino', 'amanjeev', 'JohnTitor'},
```

Comment:
either keep the `rustc-guide` maintainers or change line 206 to use `MAINTAINERS.get(tool, ())`.

Reply:
Done!